content/blogposts/2024/cody-jetbrains-6-0-16-release.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "Cody for JetBrains v6.0.16: Shortcut display updates and bug fixes"
+authors:
+  - name: Alex Isken
+    url: https://x.com/AlexanderIsken
+  - name: Justin Dorfman
+    url: https://x.com/jdorfman
+publishDate: 2024-07-25T10:00-01:00
+description: "This release includes quality-of-life updates to how shortcuts are displayed in the plugin and several bug fixes."
+tags: [blog]
+slug: "cody-jetbrains-6-0-16-release"
+published: true
+heroImage: https://storage.googleapis.com/sourcegraph-assets/blog/cody-jetbrains-6-0-16-release/cody-for-jetbrains-v6.0.16-og.jpg
```

Comment:
not loading

Reply:
`<Details>No such object: sourcegraph-assets/blog/cody-jetbrains-6-0-16-release/cody-for-jetbrains-v6.0.16-og.jpg</Details>`